lib/pages/chat_window.dart

```diff
@@ -158,7 +158,9 @@ class _ChatInboxState extends State<ChatInbox> {
     final newMessage = Message(
       message: payload["content"],
       isSender: payload["created_by_name"] == widget.userName ? false : true,
-      timestamp: DateTime.parse(payload["inserted_at"]),
+      // timestamp: DateTime.parse(payload["inserted_at"]),
+      timestamp: DateTime.now(),
```

Comment:
we should use the payload timestamp instead of the current timestamp. Please fix the parse issue. @Sanketbhavsar19